package.json

```diff
@@ -20,6 +20,7 @@
     "eslint": "eslint . --report-unused-disable-directives",
     "eslint-fix": "eslint --fix . --report-unused-disable-directives",
     "generate": "run-s generate:*",
+    "generate:distro-json-generate": "node tools/distro-json-generate.mjs",
```

Comment:
@rarkins I don't think we should generate this file on every `generate` call. So this should be reverted. WDYT?

Reply:
@Gabriel-Ladzaretti Please revert this, we will update file on demand.

Reply:
reverted